src/js/components/ScaleList.jsx

```diff
@@ -1,8 +1,22 @@
 'use strict';
 var React = require('react'),
+    connect = require('react-redux').connect,
     ContentEditable = require('./ContentEditable');
 
-var ScaleList = React.createClass({
+function mapStateToProps(reduxState, ownProps) {
+  return {
+    selected: reduxState.get('selectedMark')
+  };
+}
+
+var ScaleList = connect(
```

Comment:
Please don't feel bad; I agree with your sentiment and I appreciate your raising it!